<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1318</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1311</width>
      <height>761</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>120</y>
        <width>451</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(228,224,223);
color: rgb(0, 0, 106);
color: rgb(0, 0, 79);
color: rgb(0, 0, 127);
alternate-background-color: rgb(187, 240, 250);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;gestion des commandes&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>170</y>
        <width>541</width>
        <height>461</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>10</y>
        <width>401</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255); border-radius:20%;
color: rgb(0, 0, 0);
font: 700 16pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>        Gestion des Commandes </string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>100</y>
        <width>131</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255,255, 255); border-radius:10%;
font: 9pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);
</string>
      </property>
      <property name="text">
       <string>Statistique</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>140</y>
        <width>471</width>
        <height>521</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(252,224,187);
alternate-background-color: rgb(153, 204, 153);
font: 18pt &quot;Segoe UI&quot;;
border-radius:20%;</string>
      </property>
      <property name="title">
       <string>Commande</string>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>221</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 127);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;idproduit :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>150</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 127);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Nomproduit :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>290</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 127);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Prix :&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>360</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 127);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Quantit√® :&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_id_2">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>100</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>18</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
 border-radius:10%;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nom_2">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>160</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>18</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"> border-radius:10%;background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_prix_2">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>290</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>18</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"> border-radius:10%;background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="inputMask">
        <string notr="true"/>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_quantite_2">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>360</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>18</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"> border-radius:10%;background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>220</y>
         <width>141</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 127);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Payment :&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_payment_2">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>220</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>18</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"> border-radius:10%;background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Ajouter_2">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>470</y>
         <width>131</width>
         <height>29</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(13, 33, 79); border-radius:10%;
font: 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(71, 170, 82);
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="mettre_a_jour">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>470</y>
         <width>131</width>
         <height>29</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(13, 33, 79); border-radius:10%;
font: 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 170, 0);
background-color: rgb(71, 170, 82);
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Mettre a jour</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>670</y>
        <width>131</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(13, 33, 79); border-radius:10%;
font: 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(71, 170, 82);
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Exporter pdf</string>
      </property>
     </widget>
     <widget class="QPushButton" name="supprimer">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>670</y>
        <width>131</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(13, 33, 79); border-radius:10%;
font: 9pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(71, 170, 82);
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>640</y>
        <width>241</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Search :</string>
      </property>
      <widget class="QRadioButton" name="Radio_Nom">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>110</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="Radio_Payment">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>110</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Payment</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Search_LineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>221</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QRadioButton" name="Radio_Rien">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>20</y>
         <width>61</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Rien</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>1000</x>
        <y>640</y>
        <width>301</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Filtre :</string>
      </property>
      <widget class="QRadioButton" name="Radio_NomFiltre">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>110</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="Radio_PaymentFiltre">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>110</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Payment</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="Radio_QuantiteFiltre">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>20</y>
         <width>110</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Quantite</string>
       </property>
      </widget>
     </widget>
     <widget class="QListView" name="listView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>191</width>
        <height>491</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(13, 33, 79); border-radius:20%;
background-color: rgb(226, 226, 226);
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Sunken</enum>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>1000</x>
        <y>690</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QRadioButton" name="Radio_Asc">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>110</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Ascendant</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="Radio_Desc">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>10</y>
         <width>110</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Descendant</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="Radio_RienFilter">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>10</y>
         <width>61</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Rien</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QFrame" name="ChartFrame">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>1281</width>
        <height>681</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="page2a1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>93</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt; Avant</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>10</y>
        <width>301</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Critere  :</string>
      </property>
      <widget class="QRadioButton" name="Radio_ChartNom">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>61</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="Radio_ChartPayment">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>20</y>
         <width>91</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Payment</string>
       </property>
      </widget>
     </widget>
     <zorder>page2a1</zorder>
     <zorder>groupBox_5</zorder>
     <zorder>ChartFrame</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1318</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
